File: HabaClimate/Data/Models/Good.cs
namespace HabaClimate.Data.Models
{
    public class Good
    {
        public int Id { set; get; }
        public string Name { set; get; }
        public string ShortDesc { set; get; }
        public string LongDesc { set; get; }
        public string Img { set; get; }
        public decimal Price { set; get; }
        public bool Available { set; get; }
        public bool IsFavorite { get; set; }
        
        public int CategoryId { set; get; }
        public virtual Category Category { set; get; }
        
        public virtual Brand Brand { get; set; }
    }
}

File: ClientApp/src/app/_models/Good.ts
export interface Good {
  id: number;
  name: string;
  shortDesc: string;
  longDesc: string;
  img: string;
  price: number;
  available: boolean;
  isFavorite: boolean;
  categoryId: number;
  categoryName: string;
  brandName: string;
}
